src/jupynium/lua/helpers.lua

```diff
@@ -688,3 +688,42 @@ function Jupynium_kernel_complete_async(bufnr, code_line, col, callback)
 
   Jupynium_rpcnotify("kernel_complete_async", bufnr, true, code_line, col, callback_id)
 end
+
+function Jupynium_connect_kernel(bufnr, hostname)
+  if bufnr == nil or bufnr == 0 then
+    bufnr = vim.api.nvim_get_current_buf()
+  end
+
+  local kernel_id = nil
+  if Jupynium_syncing_bufs[bufnr] ~= nil then
+    kernel_id = Jupynium_rpcrequest("kernel_connect_info", bufnr, true)
+  end
+  if kernel_id == nil then
+    kernel_id = ""
+  end
+  local jupyter_command = "jupyter"
+  local ok, options = pcall(require, "jupynium.options")
+  if ok then
+    if type(options.opts.jupyter_command) == "string" then
+      jupyter_command = options.opts.jupyter_command
+    elseif type(options.opts.jupyter_command) == "table" then
+      jupyter_command = table.concat(options.opts.jupyter_command, " ")
+    else
+      Jupynium_notify.error { "Invalid jupyter_command type." }
+    end
+  end
+  if hostname ~= "" then
+    jupyter_command = "ssh " .. hostname .. " -t " .. jupyter_command
+  end
+  Jupynium_notify.info { "Connecting to kernel " .. kernel_id }
+  local cmd = jupyter_command .. " console --existing " .. kernel_id
+  return cmd
+end
```

Comment:
Currently, the Jupynium_connect_kernel just returns the cmd to connect to the kernel. Is this intentional? Maybe the function name should be better, like `Jupynium_get_kernel_connect_shcmd` and `Jupynium_kernel_connect_cmd`.